docs/spec/mint/01_concepts.md

```diff
@@ -0,0 +1,23 @@
+# Concepts
+
+## The Minting Mechanism
+
+The minting mechanism was designed to:
+ - allow for a market evaluation of the inflation rate
+ - optimize between market liquidity and staked supply.
+
+In order to best determine the appropriate market rate for inflation rewards, a
+moving change rate is used.  The moving change rate mechanism ensures that if
+the % bonded is either over of under the goal %-bonded, the inflation rate will
+adjust to further incentivize or incentive being bonded. Setting the goal
```

Comment:
`to further incentivize or incentive being bonded.` ...seems a bit off to me, no?

Reply:
whoops .. updated `incentivize or incentive` -> `incentivize or disincentivize`